ci/win_install_deps.cmd

```diff
@@ -66,10 +66,10 @@ mingw-w64-x86_64-cmake ^
 mingw-w64-x86_64-ninja ^
 mingw-w64-x86_64-ncurses ^
 mingw-w64-x86_64-readline ^
-mingw-w64-x86_64-python3 ^
+mingw-w64-x86_64-python ^
```

Comment:
Just curious. How did you figure out that this would help?

Reply:
On this page: https://packages.msys2.org/packages/mingw-w64-x86_64-python3
it is mentioned that `mingw-w64-x86_64-python3` is a virtual package, and is provided by `mingw-w64-x86_64-python` anyway, so they're the same thing. Same problem was fixed in #3252, but for `setuptools`, so we may as well use all of the `python` and not `python3` packages (since the latter are apparently being phased out).